src/main/java/edu/harvard/iq/dataverse/util/json/JsonLDTerm.java

```diff
@@ -48,6 +45,8 @@
     public static JsonLDTerm totalSize = JsonLDTerm.DVCore("totalSize");
     public static JsonLDTerm fileCount = JsonLDTerm.DVCore("fileCount");
     public static JsonLDTerm maxFileSize = JsonLDTerm.DVCore("maxFileSize");
+    
+    public static JsonLDTerm metadataOnOrig = JsonLDTerm.DVCore("metadataOnOrig");

```

Comment:
Ah, it seems like maybe all installations do need the new .tsv file. If so, it should be added to the script that adds them all during installation.

Reply:
As noted above, this is optional/not needed by default. If we decide to remove the functionality, this line would also go.

Reply:
Removed in a5a745d.